docs/deployment/getting-started.md

```diff
@@ -0,0 +1,180 @@
+# Deployment
+
+Server deployment is a complex area, that will depend on what kind of service you're deploying Uvicorn onto.
+
+### TL;DR
+As a general rule, you probably want to:
+
+* Run `uvicorn --reload` from the command line for local development.
+* Run `gunicorn -k uvicorn.workers.UvicornWorker` for production (non containerized).
+* Run behind Nginx for self-hosted deployments.
+* Finally, run everything behind a CDN for caching support, and serious DDOS protection.
+
+## Quick Start
+
+In the Quick Start section, we will cover how to start serving your application. The application mentioned here is any ASGI compatible application. For instance if an app that was built using a framework like `Starlette` or `FastAPI`.
+
+The following are ideal for running your application in a development environment.
+
+### Running from the command line
+
+The simplest way to run `uvicorn` is from the command line.
+
+```bash
+$ uvicorn example:app --reload --port 5000
+```
+
+The ASGI application should be specified in the form `path.to.module:instance.path`
+
+Use `--reload` to turn on auto-reloading. 
+???+ note
+    `--reload` is recommended only for local use, during the development phase.
+    
+    The `--reload` and `--workers` arguments are **mutually exclusive**.
+
+
+
+
+To see the complete set of available options, use `uvicorn --help`:
+
+<!-- :cli_usage: -->
```

Comment:
Can we just link to the `Introduction` section of the docs instead of repeating the CLI options here?

Reply:
I was thinking we add these here, and link this in introduction.🧐

Reply:
Let's start with the smallest change possible, we can always make more sweeping changes to the docs later on if it's warranted